\documentclass[hidelinks, 11pt]{article}

\title{Smartphone apps for portable \\ DNA sequencing and data analysis}
\author{Ben Zwart, Carson Zhang, Huan Yi Shen, Julia Danieli, Kamalesh Reddy, Prakriti Chhabra}

\usepackage[letterpaper,margin=1in]{geometry}
\usepackage{graphicx}
\usepackage{hyperref}
\usepackage{amsmath}
\usepackage{amssymb}
\usepackage{bm}
\usepackage{listings}
\usepackage{color}
\usepackage{tikz}
\usetikzlibrary{decorations.pathreplacing,shapes.misc}
\usepackage{hologo} % or use package holog?

\usepackage{empheq}
\usepackage{color}
%\usepackage{parallel,enumitem}
%\usepackage{textgreek}

\usepackage[T1]{fontenc}
\usepackage{textcomp}
\usepackage{graphicx}
\usepackage{caption}
\usepackage{subcaption}
\usepackage{enumitem}
\usepackage[normalem]{ulem}
\usepackage{xcolor}
\usepackage{array}

\lstset{upquote=true}

\UseRawInputEncoding
\usetikzlibrary{decorations.pathmorphing,patterns}

\definecolor{myblue}{rgb}{.92549, .98824, 0.95686}
\newlength\mytemplen
\newsavebox\mytempbox

\makeatletter
%“I don’t know if we were particularly lucky, but I really enjoyed every aspect of the summer student program: work, lectures and social life (a lot!).”

%Summer Student Programme
%How do you plan to spend your summer? How does getting involved in some of the world’s biggest experiments sound? This is more than summer work. It’s the chance to join CERN in Geneva – getting involved in the day-to-day work of our multicultural teams. Attend lectures, visit CERN facilities, take part in discussions and workshops with people who are leaders in their fields. In fact, it will be a summer like nowhere else on Earth.

%Openlab summer Student Programme
%Summer work experience in a place like nowhere else on earth: the birthplace of the world wide web! If you’re studying computer science, a great way to spend your summer could be to join the openlab summer student programme, to work on an advanced IT project and follow IT lectures specially prepared for you by experts at CERN and other institutes. Visits to the accelerators and experimental areas are also part of the programme, along with visits to external companies. Interested? Check it out and take part!r
\newcommand\mybluebox{%
\@ifnextchar[%]
{\@mybluebox}%
{\@mybluebox[0pt]}}

\def\@mybluebox[#1]{%
\@ifnextchar[%]
{\@@mybluebox[#1]}%
{\@@mybluebox[#1][0pt]}}

\def\@@mybluebox[#1][#2]#3{
\sbox\mytempbox{#3}%
\mytemplen\ht\mytempbox
\advance\mytemplen #1\relax
\ht\mytempbox\mytemplen
\mytemplen\dp\mytempbox
\advance\mytemplen #2\relax
\dp\mytempbox\mytemplen
\colorbox{myblue}{\hspace{1em}\usebox{\mytempbox}\hspace{1em}}}

\makeatother

\definecolor{dkgreen}{rgb}{0,0.6,0}
\definecolor{gray}{rgb}{0.5,0.5,0.5}
\definecolor{mauve}{rgb}{0.58,0,0.82}

\lstset{frame=tb,
language=Python,
aboveskip=3mm,
belowskip=3mm,
showstringspaces=false,
columns=flexible,
basicstyle={\small\ttfamily},
numbers=none,
numberstyle=\tiny\color{gray},
keywordstyle=\color{blue},
commentstyle=\color{dkgreen},
stringstyle=\color{mauve},
breaklines=true,
breakatwhitespace=true,
tabsize=3
}

\newcommand{\Lagr}{\mathcal{L}}

\def\changemargin#1#2{\list{}{\rightmargin#2\leftmargin#1}\item[]}
\let\endchangemargin=\endlist 

\newcommand\blfootnote[1]{%
  \begingroup
  \renewcommand\thefootnote{}\footnote{#1}%
  \addtocounter{footnote}{-1}%
  \endgroup
}

\begin{document}

\maketitle

\begin{center}
  \noindent\rule{16cm}{0.4pt}
  \section*{Abstract} %\cite{Taylor2005}}
  \noindent\rule{16cm}{0.4pt}
\end{center}

\begin{changemargin}{1cm}{1cm}
\textit{The following entails recent advancements in portable DNA sequencing and aims to answer some questions about the same.}
\end{changemargin}

\begin{center}
  \noindent\rule{16cm}{0.4pt}
  \section*{Introduction} %\cite{Taylor2005}}
  \noindent\rule{16cm}{0.4pt}
\end{center}

\begin{changemargin}{0.3cm}{0.3cm}

\noindent First, we will look at some basic biology concepts and answer some questions to better understand DNA sequencing:

\begin{changemargin}{0.9cm}{0.9cm}
  \begin{center}
    \noindent \textbf{What is DNA sequencing?}
  \end{center}
  DNA determines what proteins are synthesized and so, they play an important role in determining how an organism behaves and looks like (determines its characteristics and also charchteristics of its offspring). How are these instructions stored? They are stored as a long sequence of 4 chemicals/letters (Adenine $\rightarrow$ A, Thymine $\rightarrow$ T, Cytosine $\rightarrow$ C, and Guanine $\rightarrow$ G). DNA sequencing is the process of determining the order of these 4 chemicals, given a strand of DNA.
  \begin{center}
    \noindent \textbf{How is DNA sequenced? ("Shotgun Sequencing")}
  \end{center}
  DNA is a very long sequence of A's, T's, C's, and G's. So, the strand is first broken (how?) into smaller strands. The smaller strands are then sequenced individually (If you can sequence a shorter strand why not a longer one; and how long of a strand CAN you sequence?). Then, we create thousands of copies of each small strand (to easily identify them?). These small strands are then individually sequenced using enzymes and the four molecules; A, C, G, and T (look into mechanism). Now, somehow using the aforementioned enzymes and molecules we attain the sequences of these individual strands, these small sequences are stitched together using computers  (is this where data analysis plays a major role?) to give us \emph{the sequence of the entire genome}.
\end{changemargin}

\begin{figure}
  \centering
  \begin{subfigure}{.5\textwidth}
    \centering
    \includegraphics[scale=0.18]{1024px-DNA_Structure+Key+Labelled.pn_NoBB.png}
    \caption{3D model of helical structure of DNA}
    \label{fig:sub1}
  \end{subfigure}%
  \begin{subfigure}{.5\textwidth}
    \centering
    \includegraphics[scale=0.195]{800px-DNA_chemical_structure.svg.png}
    \caption{Demonstrates size of DNA}
    \label{fig:sub2}
  \end{subfigure}
  \caption{Structure of double-stranded DNA for reference}
  \label{fig:test}
  \end{figure}

\begin{center}
  \noindent\rule{16cm}{0.4pt}
  \section*{Portable DNA Sequencing} %\cite{Taylor2005}}
  \noindent\rule{16cm}{0.4pt}
\end{center}

There are a variety of sequencing methods, we will be looking into "Nanopore Sequencing". This method is, byfar, the most portable. There are other sequencers (Illumina HiSeq 2500 sequencer and BGI MGISEQ-2000RS sequencer for instance) that use fairly small machines (possibility of miniturization?) and other sequencing methods, but our current state of technology does not allow miniturization (yet!).
  \begin{center}
    \noindent \textbf{Nanopore Sequencing}
  \end{center}
  Since we have established that: right now, nanopore sequencing is the most portable form of sequencing; we will first look at what nanopore sequncing is. It is best explained by (as we will see later) the current pioneer in nanopore technology, Oxford Nanopore Technologies:
  \begin{changemargin}{2cm}{2cm}
    ``Nanopore sequencing is a unique, scalable technology that enables direct, real-time analysis of long DNA or RNA fragments. It works by monitoring changes to an electrical current as nucleic acids are passed through a protein nanopore. The resulting signal is decoded to provide the specific DNA or RNA sequence.'' \textsuperscript{[7]} %\footnote[1]{Hey}
  \end{changemargin}
  So, nanopore sequencing involves passing a single-stranded DNA molecule through a "protein nanopore" and measuring changes in electric current to determine the base pair (by "base calling" using \emph{Guppy}\textsuperscript{[17]}). This makes sense because we know that atoms are around $10^{-10}$m in diameter, so for a single strand of DNA (which is around 8-10 atoms wide) we have a width of  $\approx 10^{-9}$m $= 1$nm; hence nanopore! The devices used are surprisingly small, they are \textbf{really} small. \textsuperscript{[8]}

  \begin{figure}
    \begin{subfigure}{.5\textwidth}
      \centering
      \includegraphics[scale=0.2]{ont_1.jpg}
      \caption{Fongle}
    \end{subfigure}%
    \begin{subfigure}{.5\textwidth}
      \centering
      \includegraphics[scale=0.18]{ont_2.jpg}
      \caption{\textbf{MinION}}
    \end{subfigure}
    \caption{The Fongle and MinION can be used for portable analysis}
    \end{figure}

    \begin{figure}
      \begin{subfigure}{.5\textwidth}
        \centering
        \includegraphics[scale=0.4]{ont_4.png}
        \caption{GridION}
      \end{subfigure}%
      \begin{subfigure}{.5\textwidth}
        \centering
        \includegraphics[scale=0.195]{ont_5.png}
        \caption{PromethION}
      \end{subfigure}
      %\begin{changemargin}{3cm}{3cm}
      \caption{GridION and PromethION can be used for flexible, high-throughput benchtop sequencing}
      %\end{changemargin}
    \end{figure}

    \begin{figure}
      \begin{subfigure}{.5\textwidth}
        \centering
        \includegraphics[scale=0.3]{ont_6.png}
        \caption{SmidgION}
      \end{subfigure}%
      \begin{subfigure}{.5\textwidth}
        \centering
        \includegraphics[scale=0.195]{ont_7.jpg}
        \caption{Plongle}
      \end{subfigure}
      %\begin{changemargin}{3cm}{3cm}
      \caption{SmidgION and Plongle are under development and are excitingly small}
      %\end{changemargin}
    \end{figure}

    \begin{center}
      \noindent\rule{16cm}{0.4pt}
      \section*{Smartphone apps} %\cite{Taylor2005}}
      %\noindent\rule{16cm}{0.4pt}
    \end{center}

    % #### 23andMe, AncestryDNA, MyHeritage, FamilyTreeDNA and Dante Labs
    % #### Android and IOS have a couple of data analysis apps.

    \noindent\rule{16cm}{0.4pt}

    \hspace{0.4cm}

    \begin{center}
      \begin{tabular}{|c|c|c|}
        \hline
        \textbf{DNA sequencing} & \multicolumn{2}{|c|}{\textbf{DNA data analysis}} \\
        \hline \
        MinKNOW & GenoPo & iGenomics \\
        \hline
      \end{tabular}
    \end{center}
    
    \centering

    \begin{changemargin}{2cm}{2cm}
    \noindent\rule{12cm}{0.4pt}
    \begin{center}
      \textbf{DNA sequencing}
      \noindent\rule{12cm}{0.1pt}
      \begin{empheq}[box={\mybluebox[5pt][10pt]}]{equation*}
        \noindent \textbf{What mobile apps are available in this space?}
      \end{empheq}
    \end{center}

    %\emph{MinKNOW:} 
    
      Oxford Nanopore Technologies has their very own app: \textbf{MinKNOW}. As far as nanopore DNA sequencing (portable) is concerned, there are no other prominent apps (explained later).
    
    %\noindent\rule{12cm}{0.4pt}

    \begin{center}
      \begin{empheq}[box={\mybluebox[5pt][10pt]}]{equation*}
        \noindent \textbf{When were they developed?}
      \end{empheq}
    \end{center}

    %\emph{MinKNOW:}
    
      MinKNOW is a general application that existed for a while (since at least 2015) but the mobile app was formally released on \textbf{November 25th, 2020}. \textsuperscript{[9]}

    \begin{center}
      \begin{empheq}[box={\mybluebox[5pt][10pt]}]{equation*}
        \noindent \textbf{What features do they offer?}
      \end{empheq}
    \end{center}

    %\emph{MinKNOW:}\\
    As stated in ONT's website:
    \begin{changemargin}{1cm}{1cm}
      ``It carries out several core tasks, including: 
      \begin{itemize}
      \item \textbf{Data acquisition}
      \item \textbf{Real-time analysis and feedback}
      \item \textbf{Local basecalling}
      \item \textbf{Data streaming}
      \item \textbf{Progressive unblocking} \\
            %\begin{changemargin}{0cm}{0cm}
            - Increased data yield of a flow cell \\
            - Active voltage control $\to$ improved signal consistency during a run.\\
            %\end{changemargin}
      \end{itemize}
      - whilst providing device control including \textbf{selecting the run parameters}, \textbf{sample identification and tracking}, and ensuring that the platform chemistry is performing correctly to run the samples.'' \textsuperscript{[9]} \textsuperscript{[10]}
    \end{changemargin}

    It also enables users to \textbf{run group experiments} and \textbf{view/export metrics to a PDF}. Essentially, the app helps manage their devices, we shall also discuss complementary apps, iGenomics and GenoPo.

    \begin{center}
      \begin{empheq}[box={\mybluebox[5pt][10pt]}]{equation*}
        \noindent \textbf{Which apps are used most frequently by consumers?}
      \end{empheq}
    \end{center}

    %\emph{MinKNOW:}\\
    MinKNOW basically dominates everyother app (if any) as far as nanopore sequencing goes. This is mostly due to the availability of their own native devices, enabling them to modify their app as per device upgrades. Consumers are likely to use MinKNOW if they own a, say, MinION. Similar to how Android users likely use Google Chrome. Besides, sources indicate that they have patented the technology $\implies$ nanospore sequencing is intellectual property of ONT (for now?). \textsuperscript{[11]}

    \begin{center}
      \begin{empheq}[box={\mybluebox[5pt][10pt]}]{equation*}
        \noindent \textbf{Are there any advantages and limitations of these tools?}
      \end{empheq}
    \end{center}

    %\emph{MinKNOW:}\\                  
    Advantages:
    \begin{itemize}
    \item With new device rollouts, ONT can simply update MinKNOW to accomodate new devices.
    \end{itemize}
    Limitations:
    \begin{itemize}
    \item As of now, ONT does not compete with anyone when it comes to nanopore sequencing (a little with Illumina \textsuperscript{[12]}). This indicates that there is not as much a desire to release new technology.  
    \end{itemize}
  \end{changemargin}

  \newpage
  % \noindent\rule{12cm}{0.4pt}

    \begin{changemargin}{2cm}{2cm}
      \noindent\rule{12cm}{0.4pt}
        \begin{center}
          \textbf{DNA data analysis}
      \noindent\rule{12cm}{0.1pt}
          \begin{empheq}[box={\mybluebox[5pt][10pt]}]{equation*}
            \noindent \textbf{What mobile apps are available in this space?}
          \end{empheq}
        \end{center}


    There are two apps that ONT recognizes \textsuperscript{[13]}\textsuperscript{[14]} as complements to its devices and MinKNOW: \textbf{iGenomics} and \textbf{GenoPo}. They were developed without ONT's direct involvement.
    %\emph{MinKNOW:} 
    
    %\noindent\rule{12cm}{0.4pt}

    \begin{center}
      \begin{empheq}[box={\mybluebox[5pt][10pt]}]{equation*}
        \noindent \textbf{When were they developed?}
      \end{empheq}
    \end{center}    

    \begin{itemize}
      \item \emph{iGenomics} was developed by Aspyn Palatnick, Bin Zhou, Elodie Ghedin, and Michael Schatz \textsuperscript{[15]} and was published in \textbf{April 2020} (As shown in Apple's App Store).
      \item \emph{GenoPo} (formerly known as F5N \textsuperscript{[16]}) was developed by students from the \emph{University of Peradiniya} \textsuperscript{[14]}, it was released on \textbf{August 12th, 2020} (As shown in Google's Play Store).
    \end{itemize}
    %\emph{MinKNOW:}
    
    \begin{center}
      \begin{empheq}[box={\mybluebox[5pt][10pt]}]{equation*}
        \noindent \textbf{What features do they offer?}
      \end{empheq}
    \end{center}

    \emph{iGenomics}:
    \begin{itemize}
      \item \textbf{Alignment reading} (?)
      \item \textbf{Coverage profile} (?)
      \item \textbf{Call variants} (?)
      \item \textbf{Visualize results on any IOS device} (?)
      \item \textbf{AirDrop sequencing data from one device to another} (?)
    \end{itemize}
    \emph{GenoPo}:
    \begin{itemize}
      \item \textbf{Real-world applications}: GenoPo analysis happens very quickly, averaging about 27 minutes per SARS-CoV-2 sample with 5-10 mutations detected, and a consensus sequence generated. This is a small fraction of the normal turnaround time for your average SARS-CoV-2 nanopore sequencing. 
      \item \textbf{Compatibility}: Designed for compatibility with many bioinformatics analyses including sequence alignment, variant detection and DNA methylation profiling. 
      \item \textbf{Real time sequence analysis}: Allows real-time methylation calling on nanopore sequencing datasets which can also accommodate both small and large genomes. 
      \item \textbf{Control}: Features auto configured pipelines available for Arctic C Pipeline and Consensus Generation Pipeline, but also allows for manual configuration of pipelines; including a terminal environment allowing users to enter command line arguments. The terminal mode allows users to run any sub tool as well.
    \end{itemize}

    %\emph{MinKNOW:}\\
    
    \begin{center}
      \begin{empheq}[box={\mybluebox[5pt][10pt]}]{equation*}
        \noindent \textbf{Which apps are used most frequently by consumers?}
      \end{empheq}
    \end{center}

    \begin{itemize}
      \item Apple developers can choose to hide app statistics, which is unfortunately the case for iGenomics (messaged developer (Aspyn), waiting for an unlikely response).
      \item GenoPo has 1000+ installs on the Google Play Store (as shown in the Play Store). 
    \end{itemize}
  
    \begin{center}
      \begin{empheq}[box={\mybluebox[5pt][10pt]}]{equation*}
        \noindent \textbf{Are there any advantages and limitations of these tools?}
      \end{empheq}
    \end{center}
    \emph{iGenomics}:\\
    \textbf{Advantages}:
    \begin{itemize}
    \item AirDrop - DNA analysis in remote location, with/without internet
    \item User Reviews - on the Apple store, the reviews are mostly positive (limited samples)
    \item ``The novelty of this application is not in the algorithms used but rather how they have been implemented in a mobile environment.''
    \end{itemize}
    \emph{GenoPo}: \\
    \textbf{Advantages}:
  \begin{itemize} 
    \item An advantage of GenoPo is that it is able to methylate genomes of all sizes, and has been used in similar research to that we are interested in with SARS-CoV-2. 
    \item It contains manual pipeline configuration and a terminal environment which makes it very easily customizable to each Bioinformatician's needs. GenoPo is also very quickly able to complete it’s entire workflow.
  \end{itemize}
  \textbf{Limitations}:
  \begin{itemize}
    \item Although GenoPo has been popular on the Google Play store, it is unfortunately unavailable for iOS. It is safe to assume GenoPo would be used by more people if it were available for iOS devices as well. 
    \item Limited reviews of the app revealed significant flaws in the user-interface design. 11 product reviews suggest that the UI is difficult to navigate and interpret.
  \end{itemize}

\newpage

    \noindent\rule{12cm}{0.4pt}
      \begin{center}
        \textbf{DNA data analysis - comprehensive (not portable)}
        \noindent\rule{12cm}{0.1pt}
        \begin{empheq}[box={\mybluebox[5pt][10pt]}]{equation*}
          \noindent \textbf{What mobile apps are available in this space?}
        \end{empheq}
      \end{center}

      Search results for "Dna analysis", "Dna sequencing", and "Genomics":
      (\emph{Legend}: \colorbox{green}{\textbf{Active and Relevant}}, \\ 
                                    \hspace{1.5cm} \colorbox{yellow}{\textbf{Active}}/\textbf{Inactive and Interesting/Buggy}, \\
                                    \hspace{1.5cm} \colorbox{pink}{\sout{Gimmick/Irrelevant}})
    \hspace{0.8cm} \\
    \hspace{0.8cm} \\

    \centering
    \textbf{iOS}
    \begin{center}
      \begin{tabular}{|m{15em}|m{3cm}|m{3cm}|}
        \hline
        \textbf{Apps} & \textbf{Date} & \textbf{Installs} \\
        \hline 
        \colorbox{green}{\textbf{23andMe}} & 2020 & at least 10997 \\
        
        \colorbox{green}{\textbf{Ancestry}} & 2020 & at least 8067 \\
        \colorbox{green}{\textbf{MyHeritage}} & 2020 & at least 1642 \\
        \colorbox{green}{\textbf{AncestryDNA}} & 2020 & at least 376 \\
        \colorbox{green}{\textbf{iGenomics}} & 2020 & n \\
        \hline 
        \colorbox{yellow}{\textbf{Seq Analysis}} & 2020 & n \\
        \colorbox{yellow}{\textbf{Genewall}} & 2020 & n \\
        \textbf{DNA2App} & 2015 & n \\
        \textbf{DNAApp} & 2014 & n \\
        \textbf{MySequence} & 2013 & at least 2\\
        \hline 
        \colorbox{pink}{\sout{Dynamic News Analysis}} & N/A & N/A \\
        \colorbox{pink}{\sout{Data Navigation and Analysis}} & N/A & N/A \\
        \colorbox{pink}{\sout{ERHA.DNA}} & N/A & N/A \\
        \colorbox{pink}{\sout{My Skincaud}} & N/A & N/A \\
        \hline
      \end{tabular}
    \end{center}
    %\noindent\rule{12cm}{0.1pt}
    \hspace{0.8cm} \\
    \hspace{0.8cm} \\
    \centering
    \textbf{Android}
    \begin{center}
      \begin{tabular}{|m{15em}|m{3cm}|m{3cm}|}
        \hline
        \textbf{Apps} & \textbf{Date} & \textbf{Installs} \\
        \hline 
        \colorbox{green}{\textbf{Ancestry}} & 2011 & 5000000+ \\
        \colorbox{green}{\textbf{MyHeritage}} & 2011 & 5000000+ \\
        \colorbox{green}{\textbf{23andMe}} & 2018 & 1000000+ \\
        \colorbox{green}{\textbf{Genomapp}} & 2015 & 50000+ \\
        \colorbox{green}{\textbf{Genetica}} & 2020 & 1000+ \\
        \colorbox{green}{\textbf{DNA test}} & 2020 & 1000+ \\
        \colorbox{green}{\textbf{GenoPo}} & 2020 & 1000+ \\
        \hline 
        \colorbox{yellow}{\textbf{Face2Gene}} & 2016 & 50000+ \\
        \colorbox{yellow}{\textbf{CircleDNA}} & 2019 & 10000+ \\
        \textbf{dietgene} & 2020 & 10000+ \\
        \textbf{DNA+} & 2019 & 1000+\\
        \hline 
        \colorbox{pink}{\sout{My DNA}} & N/A & N/A \\
        \hline
      \end{tabular}
    \end{center}
      
%     Used to complete genome sequence of SARS-CoV-2 (human coronavirus) sequenced on nanopore device
% Taking less than 30 minutes per sample on a range of smartphones
% Entire workflow completed in an average of 27 minutes per sample detected 5-10 mutations and generated a complete consensus genome for SARS-CoV-2 in each patient
% This is a fraction of the typical turnaround time for your average SARS-CoV-2 nanopore sequencing
% The list of mutations and consensus sequences within the genome were identical to those generated by a best-practice workflow executed on a high performance computer

  \end{changemargin}

  % \begin{changemargin}{2cm}{2cm}
  %   \noindent\rule{12cm}{0.4pt}
  %   \begin{center}
  %     \textbf{GenoPo}
  %     \noindent\rule{12cm}{0.1pt}
  %     \begin{empheq}[box={\mybluebox[5pt][10pt]}]{equation*}
  %       \noindent \textbf{What mobile apps are available in this space?}
  %     \end{empheq}
  %   \end{center}
  %   hello

  %   %\emph{MinKNOW:} 
    
  %   %\noindent\rule{12cm}{0.4pt}

  %   \begin{center}
  %     \begin{empheq}[box={\mybluebox[5pt][10pt]}]{equation*}
  %       \noindent \textbf{When were they developed?}
  %     \end{empheq}
  %   \end{center}

  %   %\emph{MinKNOW:}
    
  %   \begin{center}
  %     \begin{empheq}[box={\mybluebox[5pt][10pt]}]{equation*}
  %       \noindent \textbf{What features do they offer?}
  %     \end{empheq}
  %   \end{center}

  %   %\emph{MinKNOW:}\\
    
  %   \begin{center}
  %     \begin{empheq}[box={\mybluebox[5pt][10pt]}]{equation*}
  %       \noindent \textbf{Which apps are used most frequently by consumers?}
  %     \end{empheq}
  %   \end{center}

  
  %   \begin{center}
  %     \begin{empheq}[box={\mybluebox[5pt][10pt]}]{equation*}
  %       \noindent \textbf{Are there any advantages and limitations of these tools?}
  %     \end{empheq}
  %   \end{center}
  % \end{changemargin}

  \newpage

  %\begin{center}
  %  \noindent\rule{16cm}{0.4pt}
  %  \section*{References} %\cite{Taylor2005}}
  %  \noindent\rule{16cm}{0.4pt}
  %\end{center}

  \begin{thebibliography}{15}
    %\bibitem{latexcompanion} 
    %Michel Goossens, Frank Mittelbach, and Alexander Samarin. 
    %\textit{The \LaTeX\ Companion}. 
    %Addison-Wesley, Reading, Massachusetts, 1993.
    
    %\bibitem{einstein} 
    %Albert Einstein. 
    %\textit{Zur Elektrodynamik bewegter K{\"o}rper}. (German) 
    %[\textit{On the electrodynamics of moving bodies}]. 
    %Annalen der Physik, 322(10):891–921, 1905.

    \bibitem{kurzgesagtyt} 
    Genetic Engineering Will Change Everything Forever - CRISPR, Kurzgesagt - In a Nutshell,
    \\\texttt{https://www.youtube.com/watch?v=jAhjPd4uNFY}
  
    \bibitem{tededyt} 
    How to sequence the human genome - Mark J. Kiel, TED-Ed,
    \\\texttt{https://www.youtube.com/watch?v=MvuYATh7Y74}

    \bibitem{yourgenomeyt} 
    DNA Sequencing - 3D, yourgenome,
    \\\texttt{https://www.youtube.com/watch?v=ONGdehkB8jU}

    \bibitem{wikipediadna} 
    DNA, \emph{Wikipedia},
    \\\texttt{https://en.wikipedia.org/wiki/DNA}

    \bibitem{wikipediadnaseq} 
    DNA sequencing, \emph{Wikipedia},
    \\\texttt{https://en.wikipedia.org/wiki/DNA\_sequencing}

    \bibitem{wikipediashotgun} 
    Shotgun sequencing, \emph{Wikipedia},
    \\\texttt{https://en.wikipedia.org/wiki/Shotgun\_sequencing}

    %\bibitem{wikipediadnasequencing} 
    %DNA sequencing, \emph{Wikipedia},
    %\\\texttt{https://en.wikipedia.org/wiki/DNA\_sequencing}
  
    \hspace{0.8cm}\\
    %\hspace{0.8cm}

    \bibitem{ONT1} 
    Nanopore DNA sequencing, \emph{Oxford Nanopore Technologies},
    \\\texttt{https://nanoporetech.com/applications/dna-nanopore-sequencing}

    \bibitem{ONT1} 
    Portable DNA sequencing devices, \emph{Oxford Nanopore Technologies},
    \\\texttt{https://nanoporetech.com/products}

    \hspace{0.8cm}\\
    %\hspace{0.8cm}

    \bibitem{ONT2} 
    Introducing the new MinKNOW App, \emph{Oxford Nanopore Technologies},
    \\\texttt{https://nanoporetech.com/about-us/news/introducing-new-minknow-app}

    \bibitem{ONTyt} 
    MinKNOW 2.0, a software update for nanopore sequencing, \emph{Oxford Nanopore Technologies},
    \\\texttt{https://www.youtube.com/watch?v=JuBIxHa0NrQ}

    \bibitem{ONT3} 
    Intellectual property, \emph{Oxford Nanopore Technologies},
    \\\texttt{https://nanoporetech.com/about-us/intellectual-property}

    \bibitem{genomeweb} 
    Illumina and Oxford Nanopore Settle Patent Infringement Lawsuit, \emph{genomeweb},
    \\\texttt{https://www.genomeweb.com/sequencing/illumina-and-oxford-nanopore-settle-patent \\ -infringement-lawsuit}

    \bibitem{ONT4} 
    iGenomics: Comprehensive DNA sequence analysis on your Smartphone, \emph{Oxford Nanopore Technologies},
    \\\texttt{https://academic.oup.com/gigascience/article/9/12/giaa138/6025149}

    \bibitem{ONT5} 
    Genopo: a nanopore sequencing analysis toolkit for portable Android devices, \emph{Oxford Nanopore Technologies},
    \\\texttt{https://nanoporetech.com/resource-centre/genopo-nanopore-sequencing-analysis- \\ toolkit-portable-android-devices}

    \bibitem{Schatz} 
    iGenomics, \emph{Schatz Lab},
    \\\texttt{http://schatz-lab.org/iGenomics/}

    \bibitem{Github} 
    f5n, \emph{GitHub},
    \\\texttt{https://github.com/SanojPunchihewa/f5n}

    %\hspace{0.8cm}

    \bibitem{Guppy} 
    Basecalling using Guppy, \emph{Guppy},
    \\\texttt{https://timkahlke.github.io/LongRead\_tutorials/BS\_G.html}
  
    \hspace{0.8cm}\\
    \hspace{0.8cm}


  
  \end{thebibliography}

\end{changemargin}
\end{document}